addons/snailmail/models/mail_notification.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+
+class Notification(models.Model):
+    _inherit = 'mail.notification'
+
+    notification_type = fields.Selection(selection_add=[('snailmail', 'Snailmail')])
```

Comment:
We have a task to reduce a bit notification footprint, and it includes selection keys. Some characters, for millions of notifications, make quite an heavy db. I therefore propose to reduce to 'snail' as it is normally specific enough to avoid name conflicts while staying short.

Reply:
Agree, `snail` is actually much easier to find in the code than `snailmail` due to it being a unique word.